src/main/resources/avro/bdg.avdl

```diff
@@ -727,7 +721,7 @@ record VariantEffect {
 }
 
 record DatabaseVariantAnnotation {
-  union { null, Variant } variant;
+  union { null, Genotype } genotype;
```

Comment:
This shouldn't nest a genotype. We're annotating the effect/statistics of a variant, not the called genotype.